test/e2e/support/utils/click-button.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Clicks a button based on the text on the button.
+ *
+ * @param {string} buttonText The text that appears on the button to click.
+ */
+export async function clickButton( buttonText ) {
+	const button = await page.waitForXPath( `//button[contains(text(), '${ buttonText }')]` );
```

Comment:
It might be nice to have a way to scope this button somehow. Some buttons can have very general name and have multiple instances.

Reply:
We can iterate on it later. It should be still useful in some cases as is.

I'm only worried about that because we are planning to expose shared code for e2e tests in npm package. That means we will have to maintain all those utils :)